Sources/DiscoverKit/View/ErrorView.swift

```diff
@@ -9,12 +9,23 @@ struct ErrorView: View {
     let error: Error
 
     var body: some View {
-        // TODO: add implementation
         // TODO: add proper error logging
-        Text("There was an error \(error.localizedDescription)")
-            .refreshable {
-                viewModel.load()
+        ScrollView {
+            ZStack {
+                Color(.mosoLayerColor1).ignoresSafeArea()
+                VStack(alignment: .leading, spacing: 8) {
+                    Text("Sorry, Something went wrong")
```

Comment:
based on Figma File:
```suggestion
                    Text("Sorry, something went wrong")
```